### PR TITLE
Created CONTRIBUTORS.md

### PR DESCRIPTION
**Pull request by jminor**
This is auto-generated from our GitHub commit history, and each user's GitHub user profile. If you want your name spelled differently (e.g. not "?") let me know.

### CODE REVIEW

**Comment by ssteinbach** [comment]
Ah, looks like you need to add it to the MANIFEST.in file.

**Comment by ssteinbach** [comment]
Also, does this have a script we can check in as well?  so we can update it with releases or something?

**Comment by jminor** [comment]
Yeah, where is a good place to add the script?

**Comment by ssteinbach** [comment]
How about we add a maintainers folder with that script in it?  we can put other automation there if need be.

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+This is a list of contributors to the OpenTimelineIO project, sorted alphabetically by first name.
+
+If you know of anyone missing from this list, please contact us: https://lists.aswf.io/g/otio-discussion
+
+* ? ([bashesenaxis](https://github.com/bashesenaxis))
```
Comment (jminor) [inline CONTRIBUTORS.md:5]: @bashesenaxis would you like your full name listed here?

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+* ? ([elabrosseRodeofx](https://github.com/elabrosseRodeofx))
```
Comment (jminor) [inline CONTRIBUTORS.md:6]: @elabrosseRodeofx would you like your full name listed here?

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+* ? ([hwilkes-igalia](https://github.com/hwilkes-igalia))
```
Comment (jminor) [inline CONTRIBUTORS.md:7]: @hwilkes-igalia would you like your full name listed here?

File: CONTRIBUTORS.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+* ? ([tuan-huy-truong](https://github.com/tuan-huy-truong))
```
Comment (jminor) [inline CONTRIBUTORS.md:8]: @tuan-huy-truong would you like your full name listed here?

File: CONTRIBUTORS.md (hunk 1 above)
Comment (bashesenaxis) [reply CONTRIBUTORS.md:5]: Yes please. "Bas Hesen" would be preferred.

File: CONTRIBUTORS.md (hunk 3 above)
Comment (hwilkes-igalia) [reply CONTRIBUTORS.md:7]: Yes please. It is: Henry Wilkes

thanks!

File: CONTRIBUTORS.md (hunk 2 above)
Comment (elabrosseRodeofx) [reply CONTRIBUTORS.md:6]: Yes please. I'd prefer 'Emile Labrosse' If possible.
Thanks !

**Pull request merged**